### PR TITLE
add CLI. resolves #45

### LINKED ISSUE #45: CLI
A command line interface would be awesome. If you could install the module globally and execute commands like

```
$ bsp rawtoasm a914c9f05571026bc94ad04ae03bf108d4bbe6815f1387
OP_HASH160 c9f05571026bc94ad04ae03bf108d4bbe6815f13 OP_EQUAL
```

### PR DESCRIPTION
**Pull request by JBaczuk**
```
$ bscript -h
Usage: bscript [options] [command]

Options:
  -v, --version                    output the version number
  -h, --help                       output usage information

Commands:
  assemble [options] <raw-script>
  disassemble [options] <asm>
  getopcode <word>
  getword <opcode>
  isvalid <opcode-or-word>
  isdisabled <opcode-or-word>
  describe <opcode-or-word>
```

### CODE REVIEW

File: bin/bscript.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    } else {
+      console.log(bscript.opcodes.wordIsDisabled(value))
+    }
+  })
+
+program
+  .command('describe <opcode-or-word>')
+  .action(function (value) {
+    if (!isNaN(parseInt(value))) {
+      console.log(bscript.opcodes.wordForOpcode(parseInt(value)))
```
Comment (wardlem) [inline bin/bscript.js]: For a PUSH_DATA opcode, wordForOpcode returns `undefined`. This should probably not be printed.

**Comment by JBaczuk** [comment]
>     2\. When a bad option is provided, the program should error rather than ignore the value. For example, providing `-l badliteraltype`

I agree, I think this should be implemented in the module, however, otherwise changes to the valid parameter values will have to be duplicated in the command line script as well. Could we have the bscript module throw an error that we can catch in the cli app?

**Comment by JBaczuk** [comment]
@wardlem ready for another look

**Comment by wardlem** [comment]
> Could we have the bscript module throw an error that we can catch in the cli app?

That is probably a good idea.

**Comment by wardlem** [comment]
@JBaczuk The changes look good.

**Comment by wardlem** [comment]
@JBaczuk It looks like this needs to be rebased before merging.

**Comment by JBaczuk** [comment]
rebased

**Comment by JBaczuk** [comment]
fix package-lock.json and squash

**Pull request merged**